module/VuFind/src/VuFind/Search/EDS/Options.php

```diff
@@ -783,22 +756,12 @@ public function getDefaultFilters()
     }
 
     /**
-     * Extract a component from the defaultView API property.
-     *
-     * The defaultView API property takes the form vufindSetting_ebscoSetting -- the first component
-     * of the underscore-delimited string is the view name used by VuFind (e.g. list or grid).
-     * However, for EDS only list is suggested to be used. The second component is the format
-     * requested from the EDS API (e.g. title, brief or detailed).
-     *
-     * @param int     $index   Index of part to extract from the property
-     * @param ?string $default Default to use as a fallback if the property does not contain delimited values
+     * Get default view setting.
      *
      * @return string
      */
-    protected function getDefaultViewPart(int $index, ?string $default = null): string
+    protected function getConfiguredDefaultView(): string
     {
-        $apiDefaultView = $this->getApiProperty('defaultView');
-        $viewArr = explode('_', $apiDefaultView);
-        return (count($viewArr) > 1) ? $viewArr[$index] : ($default ?? $apiDefaultView);
+        return $this->getApiProperty('defaultView');
```

Comment:
I see that I looked at this several months ago and got confused. I looked at it again today and got confused again. Do you think adding a comment might help reassure future code readers that we're not breaking the get/set parallelism with this code change?

```suggestion
        // Note that getApiProperty() will retrieve any defaultView value set by setConfiguredDefaultView().
        return $this->getApiProperty('defaultView');
```

Reply:
Yeah, it's definitely confusing and I like your change.  `getApiProperty` usually does do what it says, just not quite here.